keyboards/melgeek/mach80/readme.md

```diff
@@ -0,0 +1,15 @@
+# MelGeek Mach80
+
+![Mach80](https://www.melgeek.com/collections/pcb-plate/products/melgeek-mach80-tkl87-hotswappable-rgb-pcba)
+
+A hotswap 80% RGB Custom Mechanical keyboard.
+
+* Keyboard Maintainer: [melgeek001365](https://github.com/melgeek001365)
+* Hardware Supported: Mach80 rev1 20210819 
+* Hardware Availability: [MelGeek](https://www.melgeek.com/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```